### PR TITLE
Rename gameName to appName

### PR DESCRIPTION
**Pull request by robert-altom**

### CODE REVIEW

**Comment by robert-altom** [comment]
Ping @RobertPoienar @gomboskriszta15 @ReichEdvin

File: Assets/AltTester/AltDriver/AltDriver.cs [hunk 1]
```diff
@@ -22,13 +22,13 @@ public class AltDriver
...
+        /// Initiates AltDriver and begins connection with the instrumented Unity application through to AltServer
         /// </summary>
-        /// <param name="host">The ip or hostname  AltProxy is listening on.</param>
-        /// <param name="port">The port AltProxy is listening on.</param>
+        /// <param name="host">The IP or hostname AltServer is listening on.</param>
+        /// <param name="port">The port AltServer is listening on.</param>
         /// <param name="enableLogging">If true it enables driver commands logging to log file and Unity.</param>
         /// <param name="connectTimeout">The connect timeout in seconds.</param>
-        public AltDriver(string host = "127.0.0.1", int port = 13000, bool enableLogging = false, int connectTimeout = 60, string gameName = "__default__")
+        public AltDriver(string host = "127.0.0.1", int port = 13000, bool enableLogging = false, int connectTimeout = 60, string appName = "__default__")
```
Comment (RobertPoienar) [inline Assets/AltTester/AltDriver/AltDriver.cs:31]: Can you please document the appName parameter

**Review by RobertPoienar — CHANGES_REQUESTED** [review]
Just a little comment, the rest looks ok

File: Assets/Editor/CreateAltPrefab.cs [hunk 2]
```diff
@@ -658,10 +222,524 @@ private static bool quaternionEquality(Quaternion originalQuaternion, Quaternion
...
+            LabelRectTransform.SetParent(parent, false);
+
+            LabelRectTransform.localPosition = new Vector3(0, -50, 0);
+            LabelRectTransform.anchorMin = new Vector2(0.5f, 0.5f);
+            LabelRectTransform.anchorMax = new Vector2(0.5f, 0.5f);
+            LabelRectTransform.sizeDelta = new Vector2(400, 75);
+            LabelRectTransform.pivot = new Vector2(0.5f, 0.5f);
+
+            var LabelText = LabelGameObject.GetComponent<Text>();
+            LabelText.text = "To change the host, port or game name input a new value and press the Restart button.";
```
Comment (iulianatodoran) [inline Assets/Editor/CreateAltPrefab.cs:374]: should be `app name` here

**Comment by iulianatodoran** [comment]
In the `AltDesktopPage.cs` there are 2 references to the `game`, maybe they should be also changed with `app`
image

**Comment by iulianatodoran** [comment]
All the screenshots with the `game` and `AltProxy` name need to be updated also

**Comment by iulianatodoran** [comment]
Found that in all these files there are references to the game. should they also be changed?
image

**Comment by robert-altom** [comment]
> Found that in all these files there are references to the game. should they also be changed? image

I fixed the ones from docstings and `README.md`.

**Comment by robert-altom** [comment]
> All the screenshots in the documentation with the `game` and `AltProxy` name need to be updated also

We also need to uncomment and update all references to `AltProxy`.

**Comment by robert-altom** [comment]
> In the `AltDesktopPage.cs` there are 2 references to the `game`, maybe they should be also changed with `app` image

Fixed.

File: Assets/Editor/CreateAltPrefab.cs (hunk 2 above)
Comment (robert-altom) [reply Assets/Editor/CreateAltPrefab.cs]: Fixed.

**Review by RobertPoienar — APPROVED** [review]

**Review by iulianatodoran — APPROVED** [review]

**Pull request merged**